### DISCUSSION TITLE
Cant respond with Pong in interaction response

### DISCUSSION THREAD

**Discussion by Placeblock** [Rapptz/discord.py#7093] [Q&A] [answered]
Hey! Today i just tried out the "new" buttons.
For this i used python and a package called discord-components!
I know that this isnt an official package, but i think my problem isnt because of that.

My Goal was to respond just with Pong (1) to a interaction:
https://discord.com/developers/docs/interactions/slash-commands#interaction-response-object-interaction-application-command-callback-data-structure

But this error occurs:
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In Type: Value must be one of (4, 5, 6, 7).

But in the docs it says that 1 is for just pong? i dont want to edit the message or send a new one :D

Thanks for any help : )

**Comment by Rapptz** [comment, marked as answer]
We don't help with external libraries.

I will however mention that the `pong` response is only for the incoming webhooks -- not over the gateway. So you can't use it.

**Reply by Placeblock** [reply]
Thanks for your answer :D, but is there any way to respond without editing or sending something?
